libraryversions.toml

```diff
@@ -311,55 +310,3 @@ WINDOW_EXTENSIONS_CORE = { group = "androidx.window.extensions.core", atomicGrou
 WINDOW_SIDECAR = { group = "androidx.window.sidecar", atomicGroupVersion = "versions.WINDOW_SIDECAR" }
 WORK = { group = "androidx.work", atomicGroupVersion = "versions.WORK" }
 XR = { group = "androidx.xr", atomicGroupVersion = "versions.XR" }
-
-# JetBrains overrides
-ANNOTATION_JETBRAINS = { group = "org.jetbrains.compose.annotation-internal", atomicGroupVersion = "versions.COMPOSE", overrideInclude = [ ":annotation:annotation" ] }
-COLLECTION_JETBRAINS = { group = "org.jetbrains.compose.collection-internal", atomicGroupVersion = "versions.COMPOSE", overrideInclude = [ ":collection:collection" ] }
-COMPOSE_COMPILER = { group = "org.jetbrains.compose.compiler", atomicGroupVersion = "versions.COMPOSE" }
-COMPOSE_DESKTOP = { group = "org.jetbrains.compose.desktop", atomicGroupVersion = "versions.COMPOSE" }
-COMPOSE_MATERIAL_NAVIGATION = { group = "org.jetbrains.compose.material", atomicGroupVersion = "versions.COMPOSE_MATERIAL_NAVIGATION", overrideInclude = [ ":compose:material:material-navigation" ] }
```

Comment:
It seems `versions.COMPOSE_MATERIAL_NAVIGATION` is unused now and should be removed

Reply:
Done

Reply:
No, we should change it in the JetBrainsPublication (new class)